2-mandatory/3-stocks.js

```diff
@@ -34,9 +34,20 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
```

Comment:
Nice work 👍 
Do you really need the `count` variable? Can you think of another way to get the number of values in an array?